cinema/serializers.py

```diff
@@ -125,9 +125,9 @@
     def validate(self, attrs):
         data = super(TicketSerializer, self).validate(attrs=attrs)
         Ticket.validate_ticket(
-            attrs["row"], 
-            attrs["seat"], 
-            attrs["movie_session"].cinema_hall, 
+            attrs["row"],
+            attrs["seat"],
+            attrs["movie_session"].cinema_hall,
             ValidationError
```

Comment:
The `Ticket.validate_ticket` method is called with `ValidationError` as an argument. Verify if this is the intended use and that `validate_ticket` is designed to accept an exception class. If not, adjust the implementation accordingly.